### PR TITLE
docs: self hosting

### PR DESCRIPTION
**Pull request by n1ru4l**
Closes https://github.com/kamilkisiela/graphql-hive/issues/312

### CODE REVIEW

File: docker-compose.community.yml [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+      interval: 5s
+      timeout: 5s
+      retries: 6
+    networks:
+      - 'stack'
+    volumes:
+      - ./.hive/postgres:/var/lib/postgresql/data
+
+  clickhouse:
+    image: clickhouse/clickhouse-server:22.6.1-alpine
```
Comment (kamilkisiela) [inline docker-compose.community.yml:25]: You can use latest ClickHouse 22.9

Comment (kamilkisiela) [reply docker-compose.community.yml:25]: not sure if we have it on `main`, if not, ignore what I said

File: packages/web/docs/pages/self-hosting/get-started.mdx [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+```
+
+After downloading the file, you need to set some environment variables.
+
+<Callout>
+  **GitHub.** For receiving the `GITHUB_APP_ID` and `GITHUB_APP_PRIVATE_KEY` values follow the [Creating a GitHub
+  App](https://docs.github.com/en/developers/apps/building-github-apps/creating-a-github-app) documentation.
+</Callout>
+
+<Callout>
```
Comment (kamilkisiela) [inline packages/web/docs/pages/self-hosting/get-started.mdx:43]: Are you sure slack is required? I think if a user does not click "Connect Slack" in org settings then it's fine and everything works.

File: packages/web/docs/pages/self-hosting/get-started.mdx [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+  **Slack.** You need to create a new Slack app for receiving the `SLACK_CLIENT_ID` and `SLACK_CLIENT_SECRET` values.
+  [You can create a Slack app here.](https://api.slack.com/apps/)
+</Callout>
+
+<Callout>
+  **Docker Tag.** Docker images are built and published for each single commit within the GraphQL Hive repository and
+  tagged with the corresponding commit sha. You can also find all available images on the [Hive Docker Registry
+  Overview](https://github.com/kamilkisiela?tab=packages&repo_name=graphql-hive). We recommend using one of the latest
+  versions. Right now we do not have a stable release strategy hey. However, every commit on the `main` branch can be
```
Comment (kamilkisiela) [inline packages/web/docs/pages/self-hosting/get-started.mdx:52]:
```suggestion
  versions. Right now we do not have a stable release strategy yet. However, every commit on the `main` branch can be
```

**Review by kamilkisiela — APPROVED** [review]

File: packages/web/docs/pages/self-hosting/get-started.mdx (hunk 2 above)
Comment (n1ru4l) [reply packages/web/docs/pages/self-hosting/get-started.mdx:43]: we can make it optional afterwards - though it shouldnt block this PR from advancing?

**Pull request merged**